src/routing/http-routing-v1.md

```diff
@@ -93,6 +93,56 @@ The client SHOULD be able to make a request with `Accept: application/x-ndjson`
 
 Each object in the `Providers` list is a record conforming to a schema, usually the [Peer Schema](#peer-schema).
 
+### `PUT /routing/v1/providers`
```

Comment:
TODO: look into HTTP semantics for `PUT` and `POST`.

Quick brainstorm: I think the way we do announcements here is closer to `POST`  (we are not replacing old ones, old ones, if they exist, will expire according to TTL set in announcement). This means we should switch to `POST` here to avoid confusing web people, who have certain expectations around `PUT` vs `POST`

Reply:
Using POST is sensible.

For example if nodeA and nodeB ask nodeC to publish provider records on their behalf for the same CID at the same time, if they send different providers in the request, `PUT` could imply a last-update-wins style overwrite of the underlying resource when in reality both sets of provider records should be published.

Reply:
Thanks! I've switched `/routing/v1/providers` and `/routing/v1/peers` to POST in 7b32f53.